server/helpers/generateToken.js

```diff
@@ -0,0 +1,16 @@
+import jwt from 'jsonwebtoken';
+import { config } from 'dotenv';
+
+config();
+
+/**
+ * @name generateToken
+ * @param {string} payload
+ * @param {string} expiresIn
+ * @return {string} token
+ */
+const generateToken = (payload, expiresIn) => (!expiresIn
```

Comment:
Is it not better to have a default expiresIn and remove the ternary operator?

Reply:
aren't we also have the expires in set in our environment?

Reply:
The purpose of not setting the expiresIn in environment variables is simply because the expiration time may vary for different occasions moving on I'll set a default expiresIn and set  it in environment